web/src/containers/map/lib/Pois.tsx

```diff
@@ -44,16 +47,22 @@ const ClusterMarkerGlobalStyles = createGlobalStyle`
 			display: inline;
 			padding: 2px 6px;
 			border-radius: 99px;
-			background-color: #1BBD9D;
 			text-align: left;
 		}
+		
+		&--green p {
```

Comment:
Huh hoe werkt dit? Nooit eerder gezien die syntax.

Reply:
`&` neemt de classname over, hetzelfde als in Sass. Normaal zou ik er een prop van maken maar aangezien dit in een library gebeurt moest ik dit zo inbouwen.